### DISCUSSION TITLE
Elasticsearch 8 compatibility?

### DISCUSSION THREAD

**Discussion by cogliano** [DigitalPebble/storm-crawler#1005] [Q&A]
I ran the Kibana upgrade assistant and it is telling me some StormCrawler indexes are not compatible with Elasticsearch 8. Specifically, the metrics and status indexes give the error, "Date fields use deprecated camel case formats".

Has this been addressed in the latest StormCrawler code?

Thanks.

**Comment by jnioche** [comment]
As you can see in https://github.com/DigitalPebble/storm-crawler/blob/master/external/elasticsearch/archetype/src/main/resources/archetype-resources/ES_IndexInit.sh the date fields use camel case. 
This should not be hard to fix, feel free to contribute a PR if you can.

**Reply by jnioche** [reply]
The upgrade assistant might treat the camel case issue as an error  but in practice you should be able to get SC to run on ES8, thanks to https://github.com/DigitalPebble/storm-crawler/issues/962.
